pkg/model/crd.go

```diff
@@ -94,6 +94,12 @@ type CRD struct {
 	// ShortNames represent the CRD list of aliases. Short names allow shorter
 	// strings to match a CR on the CLI.
 	ShortNames []string
+	// SpecIdentifierField is the original SDK member name of the identifier
+	// spec field
+	SpecIdentifierField *string
+	// IsAdoptable determines whether the resource should be accepted by the
+	// adoption reconciler
+	IsAdoptable bool
```

Comment:
I'd prefer to have a `CRD.IsAdoptable()` method and not cache this information as a bool field... the other CRD methods that return config-driven attributes work this way and I don't see a need to optimize for performance in the code generator in this way. Same for `GetSpecIdentifierField()`... just rename it to `SpecIdentifierField` and get rid of the CRD.SpecIdentifierField field...